hail/python/hailtop/batch_client/aioclient.py

```diff
@@ -358,19 +357,21 @@ def create_job(self, image, command, env=None, mount_docker_socket=False,
         batch = Batch(self._client, b['id'], b.get('attributes'))
 
         specs = []
+        runner = AsyncThrottledGather(2)
```

Comment:
why do we need this change?

Reply:
I guess it could be a class attribute as well. I thought it was clearer if it was defined as a builder of sorts near the code that it's going to gather.